Sources/API/Debug.swift

```diff
@@ -7,26 +7,78 @@ public class Debug {
 
 private let pretty = Pretty()
 
-// MARK: Print to console
+// MARK: Standard API
 
 extension Debug {
-    public static func p<T>(_ target: T, debug: Bool = false) {
-        print(pString(target, debug: debug))
+    /// Output `target` to console.
+    /// - Parameter target: target
+    /// - Returns: String that same of output console.
```

Comment:
```suggestion
    /// - Returns: String that is the same as output console.
```

Reply:
That clause needs a verb in it. 
And "same" is used as adverb ahead of noun, or pronoun. This suggestion adopts the latter usage.